File: xamarinmvvmcross/PropertyCross.Core/Domain/Model/RecentSearch.cs

using PropertyCross.Core.Domain.SearchItem;

namespace PropertyCross.Core.Domain.Model
{
  public class RecentSearch
  {
    public SearchItemBase Search { get; set; }

    public int ResultsCount { get; set; }

    public RecentSearch ()
    {
    }

    public RecentSearch(SearchItemBase searchItem, int resultsCount)
    {
      Search = searchItem;
      ResultsCount = resultsCount;
    }
  }
}

File: neomad/src/com/propertycross/neomad/model/search/RecentSearch.java
package com.propertycross.neomad.model.search;

/**
 * @author Neomades
 */
public class RecentSearch {

	private Search search;

	private int count;

	public RecentSearch() {
	}

	public RecentSearch(Search searchType, int count) {
		this.search = searchType;
		this.count = count;
	}

	public Search getSearch() {
		return search;
	}

	public void setSearch(Search searchType) {
		this.search = searchType;
	}

	public int getCount() {
		return count;
	}

	public void setCount(int count) {
		this.count = count;
	}
}
